src/auth/auth.ts

```diff
@@ -527,8 +526,9 @@ authStore.subscribe(
   withErrorIgnoring(async (state: AuthState, oldState: AuthState) => {
     if (!oldState.termsOfService.permitsSystemUsage && state.termsOfService.permitsSystemUsage) {
       if (window.Appcues) {
-        window.Appcues.identify(userStore.get().terraUser.id!, {
-          dateJoined: parseJSON((await Ajax().User.firstTimestamp()).timestamp).getTime(),
+        const { terraUser, samUser } = userStore.get();
+        window.Appcues.identify(terraUser.id!, {
+          dateJoined: samUser.registeredAt!.getTime(),
```

Comment:
Ohhhh i looked at the liquibase changelog, I guess we set the created at date to epoch and don't initialize the registered at date, glad you mentioned it because I misremembered that sigh

Reply:
I guess I'm not sure how best to handle the case where there's no date. Without knowing what appcues is expecting I don't feel comfortable sending through something like `null` or `undefined`, and if we don't send anything the NPS might just show up all the time which doesn't sound ideal